### PR TITLE
Webkul - Misc updates

### PR DESCRIPTION
**Pull request by dhiraj-toptal**

### CODE REVIEW

**Comment by potoos** [comment]
@ynaxonBalance Can you please review?

File: Helper/Data.php [hunk 1]
```diff
@@ -36,22 +32,13 @@ public function __construct(
     public function getVendorId($sellerId)
     {
         $balancepay_vendor_id = '';
-        $storeId = $this->data->getCurrentStoreId();
         $collection = $this->sellerFactory->create()
             ->getCollection()
+            ->addFieldToSelect('balance_vendor_id')
             ->addFieldToFilter('is_seller', \Webkul\Marketplace\Model\Seller::STATUS_ENABLED)
```
Comment (ynaxonBalance) [inline Helper/Data.php:38]: import namespace (use) at beginning of file

File: Model/AbstractRequest.php [hunk 2]
```diff
@@ -231,7 +231,9 @@ protected function getLinesParams(Quote $quote, $totalShippingAmount = 0)
                 foreach ($quoteItem->getChildren() as $child) {
                     $variationId = $child->getProductId();
                     $child->getProduct()->load($child->getProductId());
```
Comment (ynaxonBalance) [inline Model/AbstractRequest.php:233]: pass `$variationId` to load method instead of calling getProductId again

File: Model/BalancepayMethod.php [hunk 3]
```diff
@@ -389,11 +397,13 @@ public function capture(InfoInterface $payment, $amount)
             $balanceVendorId = null;
 
             foreach ($orderItems as $item) {
-                $balanceVendorId = $helper->getBalanceVendors($item->getProductId());
+                $_balanceVendorId = $this->helper->getBalanceVendors($item->getProductId());
```
Comment (ynaxonBalance) [inline Model/BalancepayMethod.php:400]: what is the difference between `$_balanceVendorId` and `$balanceVendorId`? please try to come up with more descriptive variable name.

Comment (ynaxonBalance) [reply Model/BalancepayMethod.php:400]: Typo: "...Please **cleate** a separate invoice for each"

File: Model/Request/Checkout.php [hunk 4]
```diff
@@ -203,13 +210,14 @@ protected function getLineItemsParams(Quote $quote)
             }
             $variationId = $quoteItem->getProductId();
             $quoteItem->getProduct()->load($quoteItem->getProductId());
```
Comment (ynaxonBalance) [inline Model/Request/Checkout.php:212]: use `pass $variationId to load method instead of calling getProductId again

**Review by ynaxonBalance — COMMENTED** [review]
Hey Dhiraj, left some comments

File: Model/BalancepayMethod.php (hunk 3 above)
Comment (dhiraj-toptal) [reply Model/BalancepayMethod.php:400]: These variables were actually used in the previous module, I kept them as it is and added new logic around them. 
I am going to update those variables and give more descriptive name, Its confusing as well to understand the logic.

**Pull request merged**